lib/Standupguy.rb

```diff
@@ -176,25 +176,45 @@ class Item
     attr_accessor :data
 
     def method_missing(method, *args)
-      return unless [:description=, :name=, :date=].include?(method)
+      return unless [:description=, :name=, :date=, :ticket_id=].include?(method)
       method = method.to_s.chop.to_sym
       @data[method] = args.first
     end
 
+    def self.continue?
+      puts <<-EOS 
```

Comment:
trailing whitespace will be fixed. This isn't a rails project so I can write to STDOUT